main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+)
+
+func main() {
+	registry := os.Getenv("REGISTRY")
+	if registry == "" {
+		log.Fatalf("could not get registry\n")
+	}
+
+	registryOrganisation := os.Getenv("REGISTRY_ORGANISATION")
```

Comment:
Maybe a struct instead of prefixing everything with `registry` or drop the prefix at all.